### PR TITLE
creation of components for the HomePage

### PR DESCRIPTION
**Pull request by dderancourt**
creation of a few components to ease starting of the project

### CODE REVIEW

**Review by laurentmahieu — DISMISSED** [review]

File: frontend/src/components/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ListCar from "./ListeCar";
+import NavBar from "./NavBar";
+import Historic from "./Historic";
+
+function HomePage() {
+  return (
+    <div>
+      <ListCar />
+      <NavBar />
```
Comment (LoicBrassart) [inline frontend/src/components/HomePage.jsx]: Shouldn't the `<NavBar>` be present in every page ? If so, I think it should be called from the `<App>` component

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

File: frontend/src/components/HomePage.jsx (hunk 1 above)
Comment (dderancourt) [reply frontend/src/components/HomePage.jsx]: review done

**Review by LoicBrassart — APPROVED** [review]
👌

File: frontend/src/components/HomePage.jsx (hunk 1 above)
Comment (laurentmahieu) [reply frontend/src/components/HomePage.jsx]: we don't really want to see the navbar on every page ex Login PAGE, chargement page, confirmationPage(new car, new km)...
so I think it should be in a child of <App>

**Review by laurentmahieu — APPROVED** [review]

**Pull request merged**